R/create_figures_doc.R

```diff
@@ -17,12 +17,31 @@ create_figures_doc <- function(resdir = NULL,
                                year = NULL,
                                subdir = NULL,
                                include_all = TRUE) {
+
   model <- match.arg(model, several.ok = FALSE)
 
+  # import pre-written captions and alt text
   captions_alttext <- utils::read.csv(
     system.file("resources", "captions_alttext.csv", package = "asar")
   )
 
+  # import converted model output
+  output <- utils::read.csv(
+    paste0(resdir, "/", model_results, ".csv")
```

Comment:
please use `file.path()` or `fs::path()` rather than hard-coding the file separator which is operating-system specific.